apps/src/code-studio/assets/SelectStartAnimations.jsx

```diff
@@ -51,7 +51,7 @@ export default class SelectStartAnimations extends React.Component {
   addAnimationToList = animation => {
     const key = createUuid();
 
-    let updatedOrderedKeys = [key].concat(this.state.orderedKeys);
+    let updatedOrderedKeys = this.state.orderedKeys.concat([key]);
```

Comment:
This should be a `const` instead of a `let`. It's never re-assigned. Actually, eyeballing this file there are several variables which are `let`s that should be `const`.

Also, not that it's important, but I would've probably just skipped the temporary variable entirely and set it straight into the state:
```
this.setState({ orderedKeys: [...this.state.orderedKeys, key] });
```

(the spread operator vs concat is almost entirely stylistic. spread is more "modern", concat is "old", though I think concat still tends to perform better with bigger arrays)

Reply:
Good catch - it's also interesting that our linter didn't catch this? I believe we enforce the `prefer-const` rule, but maybe only in TypeScript files?

Reply:
Thanks! I like going straight into state.